packages/server/graphql/mutations/inviteToTeam.ts

```diff
@@ -80,12 +82,24 @@ export default {
         const user = users.find((user) => user.email === email)
         return !(user && user.tms && user.tms.includes(teamId))
       })
+
+      // filter out invitees that aren't approved by the org
+      const approvalErrors = await Promise.all(
+        newInvitees.map((email) => {
+          return getIsEmailApprovedByOrg(email, orgId, dataLoader)
+        })
+      )
+      const newAllowedInvitees = newInvitees
```

Comment:
if my org is locked down to only inviting people with the company.co domain, then if i invite someone by email that doesn't match that domain, it'll get filtered out